Content.Client/Actions/ActionsSystem.cs

```diff
@@ -152,7 +128,7 @@ public override void UpdateAction(EntityUid? actionId, BaseActionComponent? acti
             if (!ResolveActionData(actionId, ref action))
                 return;
 
-            action.IconColor = action.Charges < 1 ? action.DisabledIconColor : action.OriginalIconColor;
+            action.IconColor = _charges.GetCurrentCharges(actionId.Value) < 1 ? action.DisabledIconColor : action.OriginalIconColor;
```

Comment:
If something doesn't have a limited charges component, like Scream or Combat Mode, this makes those actions start greyed out.